src/main/kotlin/me/lensvol/blackconnect/config/BlackConnectSettingsPanel.kt

```diff
@@ -21,6 +23,7 @@ class BlackConnectSettingsPanel(project: Project) : JPanel() {
     private val configSections = listOf(
         SaveTriggerSection(project),
         ConnectionSection(project),
+        LocalDaemonSection(project),
```

Comment:
"Connection Settings" and "Local Instance" sections seem to be "either/or". I get it that the current structure of the settings with sections doesn't make it easy to introduce this logic, but I'd suggest refactorings settings code at some later time. It would be helpful from both from the code prespective (using Kotlin UI DSL) and from the UI logic and appearance perspective. Perhaps I could help with it.

Reply:
Actually, they are meant to be used together as in "Connect to Local Instance". But I agree, it can use some deduplication and refactoring.

Reply:
Yes, I agree there there is some duplication (host:port), but there is no "Check connection" in "Local Instance".

Reply:
"Run / Stop" conceptually act like it.

Reply:
Closing this conversation as resolved. It seems we both think that this UI could be refactored later from the UX perspective.